package.json

```diff
@@ -40,12 +40,15 @@
     "@babel/plugin-transform-runtime": "^7.8.3",
     "@babel/preset-env": "^7.8.4",
     "@babel/preset-react": "^7.8.3",
+    "@commitlint/cli": "^8.3.5",
+    "@commitlint/config-conventional": "^8.3.4",
     "@semantic-release/changelog": "^5.0.0",
     "@semantic-release/git": "^9.0.0",
     "babel-eslint": "^10.0.3",
     "babel-loader": "^8.0.6",
     "babel-plugin-transform-object-rest-spread": "^6.26.0",
     "clean-webpack-plugin": "^3.0.0",
+    "conventional-changelog-angular": "^5.0.6",
```

Comment:
Why are you installing this explicitly? This already gets installed as part of `@commitlint` packages

Reply:
ohhh thats why package.locked didnt change....

Reply:
wait,are you sure? it didnt add it for me. though I didnt install @commitlint. I installed cli and config and then this.